packages/synapse-react-client/src/synapse-queries/dataaccess/useAccessRequirements.ts

```diff
@@ -163,24 +166,65 @@ export function useCreateLockAccessRequirement(
   })
 }
 
+function getAccessRequirementStatusQueryOptions<
+  T extends
+    | AccessRequirementStatus
+    | ManagedACTAccessRequirementStatus = AccessRequirementStatus,
+>(
+  keyFactory: KeyFactory,
+  accessToken: string,
+  accessRequirementId: string,
+): //UseQueryOptions<T, SynapseClientError> // for @tanstack/react-query v5
+{ queryKey: QueryKey; queryFn: QueryFunction<T> } {
+  return {
+    queryKey:
+      keyFactory.getAccessRequirementStatusQueryKey(accessRequirementId),
+    queryFn: () =>
+      SynapseClient.getAccessRequirementStatus<T>(
+        accessToken,
+        accessRequirementId,
+      ),
+  }
+}
+
 export function useGetAccessRequirementStatus<
   T extends
     | AccessRequirementStatus
     | ManagedACTAccessRequirementStatus = AccessRequirementStatus,
 >(
   accessRequirementId: string,
+  options?: Partial<UseQueryOptions<T, SynapseClientError>>,
+) {
+  const { accessToken, keyFactory } = useSynapseContext()
+
+  const queryOptions = getAccessRequirementStatusQueryOptions<T>(
+    keyFactory,
+    accessToken!,
+    accessRequirementId,
+  )
+
+  return useQuery(queryOptions.queryKey, queryOptions.queryFn, options)
+}
+
+export function useGetAccessRequirementStatuses<
```

Comment:
Add `useQueries` variant for fetching AR status, so we can fetch more than one with one hook (there is no backend service for fetching status in multiple ARs)